squirrel.worker/src/test/java/org/dice_research/squirrel/fetcher/manage/FetcherDummy4Tests.java

```diff
@@ -0,0 +1,61 @@
+package org.dice_research.squirrel.fetcher.manage;
+
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+import org.dice_research.squirrel.fetcher.Fetcher;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FetcherDummy4Tests implements Fetcher {
```

Comment:
Please add javadoc comments to document the aim of this class, its workflow and especially the maining of the pre and post flag attributes.

I would even recommend to rename the pre and post flags since their names are not self explaining.

Reply:
@MichaelRoeder Successfully added the Javadoc comments and renamed the pre and post flag. Committed these changes to Testcase branch in abhihc/Squirrel (https://github.com/abhihc/Squirrel/commit/a0770cc4169da3057be9d947b540723018de8d65). Please let me know if I can push the code to abhihc/squirrel master so that reflects the code changes on this pull request.

Reply:
1) If you push a commit to the master branch, it will be automatically added to this PR. So no further actions are needed :wink: 
2) When you are writing a Javadoc comment, it might be worth to check how it will be shown to other users (your IDE should offer you various ways to see the Javadoc comment of a class). Please note that for example new line characters are removed. In your case, it means that all the text is written directly after each other (which is ok). However, you should make sure that your writing is correct, e.g., that you have ended the sentences with a point. Otherwise, the comment might be confusing for others :wink:

Reply:
@MichaelRoeder Please let me know if I can insert point and commit again.